pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/dedicated-project-admin.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    authorization.openshift.io/system-only: 'true'
+  name: dedicated-project-admin
+rules:
+- apiGroups:
+  - ''
+  attributeRestrictions: null
+  resources:
+  - limitranges
+  - resourcequotas
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ''
+  - network.openshift.io
+  attributeRestrictions: null
+  resources:
+  - egressnetworkpolicies
```

Comment:
Any idea why a dedicated project admin can crud egressnetworkpolicies but not a dedicated cluster admin?